src/App.vue

```diff
@@ -1,36 +1,28 @@
 <template>
   <v-app id="inspire">
     <v-app-bar class="elevation-1" app clipped-right>
-      <v-toolbar-title @click="goHome" class="logo"
-        >NimbusImage</v-toolbar-title
-      >
+      <v-toolbar-title @click="goHome" class="logo" title="NimbusImage home">
+        <img src="/img/icons/NimbusImageIcon.png" alt="Icon" class="logo-icon" />
+      </v-toolbar-title>
       <bread-crumbs />
       <v-spacer />
       <v-btn
         color="primary"
         class="ml-4"
         :to="{ name: 'newdataset' }"
         :disabled="!store.isLoggedIn"
+        title="Upload a new dataset"
       >
         Upload Data
       </v-btn>
-      <user-menu class="ml-4" />
-      <v-btn text icon class="ml-4" @click="dark = !dark">
```

Comment:
You can remove the getter and the setter for `dark`

Reply:
Sorry, I meant that you removed the only part of the code that uses the variable "dark"
So you can remove the function `get dark` (the getter of dark) and `set dark` (the setter of dark) in this file
`dark` is a special variable that is computed, you can use it as a normal variable but each time you need to get its value, the getter is used (with caching) and each time you need to set it, the setter is used

Reply:
Ahhh, okay, I see, I didn't see those functions in there. Deleted!